src/custom/components/Web3Status/Web3StatusMod.tsx

```diff
@@ -230,6 +243,17 @@ export function Web3StatusInner({
         <Text>{error instanceof UnsupportedChainIdError ? <Trans>Wrong Network</Trans> : <Trans>Error</Trans>}</Text>
       </Web3StatusError>
     )
+  } else if (thereWasAProvider) {
+    return (
+      <Web3StatusConnected pending={true} clickable={false}>
```

Comment:
ubernit: can just pass pending like so:
```suggestion
      <Web3StatusConnected pending clickable={false}>
```